shell/platform/windows/window_win32_unittests.cc

```diff
@@ -84,5 +84,27 @@ TEST(MockWin32Window, KeyDownPrintable) {
   window.InjectWindowMessage(WM_CHAR, 65, lparam);
 }
 
+TEST(MockWin32Window, KeyDownWithCtrl) {
+  MockWin32Window window;
+
+  // Simulate CONTROL pressed
+  BYTE keyboard_state[256];
+  memset(keyboard_state, 0, 256);
+  keyboard_state[VK_CONTROL] = -1;
+  SetKeyboardState(keyboard_state);
+
+  LPARAM lparam = CreateKeyEventLparam(30);
+
+  // Expect OnKey, but not OnText, because Control + Key is not followed by
+  // WM_CHAR
+  EXPECT_CALL(window, OnKey(65, 30, WM_KEYDOWN, 0, false, true)).Times(1);
+  EXPECT_CALL(window, OnText(_)).Times(0);
```

Comment:
Can you check if this test fails without the change? As far as I remember, `EXPECT_CALL.Times` only checks if this many calls have been called, but not extra calls.

Reply:
I'm not sure I understand. Without what change? This is a compltely new test. It should result in OnKey called once, and no OnText call.

Reply:
Without your fix to `window_win32.cc`. Basically, whether this test fails on master.

Reply:
Yes, the test fails because `OnKey` is not invoked on the mock without the change. It waits for WM_CHAR, which never comes.

Reply:
Got it. Thanks for confirming.